examples/bigcommerce/lib/bigcommerce/bigcommerce.ts

```diff
@@ -0,0 +1,195 @@
+const STORE_NAME = process.env.BIGCOMMERCE_STORE_NAME
+const ACCESS_TOKEN = process.env.BIGCOMMERCE_ACCESS_TOKEN
+
+const getBearerToken = async () => {
+  const A_WEEK_FROM_NOW = Math.floor(
+    new Date(new Date().getTime() + 1000 * 60 * 60 * 24 * 7).getTime() / 1000,
+  )
+
+  const response = await fetch(
+    `https://api.bigcommerce.com/stores/${process.env.BIGCOMMERCE_STORE_HASH}/v3/storefront/api-token`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Auth-Token': ACCESS_TOKEN ?? '',
+      },
+      body: JSON.stringify({
+        channel_id: 1,
+        expires_at: A_WEEK_FROM_NOW,
+        allowed_cors_origins: ['https://makeswift-example.mybigcommerce.com'],
+      }),
+    },
+  ).then(a => a.json())
```

Comment:
Since this in an `async` function I would suggest we use the `await` keyword instead of `.then`